build/testdata/bundles/wordpress/porter.yaml

```diff
@@ -42,5 +42,6 @@ install:
 uninstall:
 - helm:
     description: "Uninstall Wordpress"
-    name:
-      source: bundle.parameters.wordpress-name
+    purge: true
+    releases:
+      - "source: bundle.parameters.wordpress-name"
```

Comment:
Ooh, this works?  Handy to know...

Reply:
When you are in the yaml list, you need to put quotes around `"source: bundle...."` to make it work. But yeah it's neat!